doc/contributing.txt

```diff
@@ -0,0 +1,36 @@
+==============================
+ Contributing to PRoot / CARE
+==============================
+
+-----------------------------------------------------------
+An introduction to contributing to PRoot / CARE development
+-----------------------------------------------------------
+
+The PRoot Team welcomes, and depends, on contributions from students and
+collaborators in the open source and academic communities.
+Contributions can be made in a number of ways, a few examples are:
+
+- Code patches via pull requests
+- Documentation improvements
+- Bug reports and patch reviews
+
+Reporting an Issue
+==================
+
+Please include as much detail as you can. Let us know your platform and relevant
+version numbers for the project in question. If get an error please include the
+full error and traceback.
+
+Submitting Pull Requests
+========================
+
+Once you are happy with your changes or you are ready for some feedback, push
+it to your fork and send a pull request. For a change to be accepted it will
+most likely need to have tests and documentation if it is a new feature.
```

Comment:
New features **require** additional tests in order to be included in a future release.